### DISCUSSION TITLE
.dx files too large for 3Dmol.js

### DISCUSSION THREAD

**Discussion by akampen** [Electrostatics/apbs#279] [Q&A] [answered]
Hi devs,

I'd like to display the results of APBS in my webapp that uses 3Dmol.js, but I'm finding that the 100+ MB .dx output file is too large to reasonably be used in this way. I'm running APBS on small molecules, so I don't think it's an issue of molecule size. [This APBS paper](https://ccom.ucsd.edu/~mholst/pubs/dist/JESM17a.pdf) suggests compatibility with 3Dmol.js, but doesn't go into details. Anyone have tips on pairing 3Dmol.js with this tool? Or more generally, is there a way to configure the output so that it isn't quite so large?

Thanks in advance!

**Comment by sobolevnrm** [comment, marked as answer]
You should be able to use a combination of [dime](https://apbs.readthedocs.io/en/latest/using/input/old/elec/dime.html) and [fglen](https://apbs.readthedocs.io/en/latest/using/input/old/elec/fglen.html) to control the size of the output.

You can also see some examples of our APBS web server code at <https://github.com/Electrostatics/apbs-web/>.

Thank you
